src/styles/container.scss

```diff
@@ -0,0 +1,9 @@
+.container {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  margin: 50px auto 0;
+  padding: 0 120px;
+  box-sizing: content-box;
```

Comment:
You must use global reset styles in the individual file "reset.css":
`* {`
`  margin: 0;`
`  padding: 0;`
`  box-sizing: content-box;`
`}`